src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchQueryHelper.java

```diff
@@ -34,7 +35,12 @@ public ElasticsearchQueryHelper(ApplicationParams applicationParams,
 
     public ElasticsearchRequest validateAndConvertRequest(String jsonSearchRequest) {
         rejectBlackListedQuery(jsonSearchRequest);
-        JsonNode searchRequestNode = objectMapperService.convertStringToObject(jsonSearchRequest, JsonNode.class);
+        JsonNode searchRequestNode;
+        try {
+            searchRequestNode = objectMapperService.convertStringToObject(jsonSearchRequest, JsonNode.class);
+        } catch (Exception ex) {
+            throw new BadRequestException("Request requires correctly formatted JSON");
+        }
```

Comment:
Can you append original exception message here (ex.getMessage)

Reply:
Updated to catch Service Exceptions

Reply:
Updated to append exception message from ServiceException